app/src/main/scala/io/codegalaxy/app/info/ListItemNavIcon.scala

```diff
@@ -15,19 +15,22 @@ object ListItemNavIcon extends FunctionComponent[ListItemNavIconProps] {
   protected def render(compProps: Props): ReactElement = {
     implicit val theme: Theme = useTheme()
     val props = compProps.wrapped
-    
     <.View(^.rnStyle := styles.statsContainer)(
-      if (props.progress > 0) <.>()(
-        if (props.showLabel) Some(<.Text(^.rnStyle := styles.statsLabel)("Open"))
-        else None,
-        <.View(^.rnStyle := styles.statsProgress)(
-          <.Text(^.rnStyle := themeTextStyle)(s"${props.progress}")
-        )
-      )
-      else <.>()(
-        if (props.showLabel) Some(<.Text(^.rnStyle := styles.statsLabel)("Start"))
-        else None,
-        <.SvgXml(^.rnStyle := styles.startSvg, ^.xml := startSvgXml)()
+      <.>()(
+        Some(props.showLabel)
+          .filter(identity)
+          .map { _ =>
+            <.Text(^.rnStyle := styles.statsLabel)(
+              if (props.progress > 0) "Continue" else "Start"
```

Comment:
I guess `Open` was used previously, since on smaller devices screens `Continue` is too long and breaks layout. I remember testing this on iPhone 5s, for example.